core/cmd/client.go

```diff
@@ -194,7 +195,7 @@ func (n ChainlinkAppFactory) NewApplication(cfg config.GeneralConfig) (chainlink
 		}
 	}
 
-	eventBroadcaster := pg.NewEventBroadcaster(cfg.DatabaseURL(), cfg.DatabaseListenerMinReconnectInterval(), cfg.DatabaseListenerMaxReconnectDuration(), appLggr, appID)
+	eventBroadcaster := services.NewEventBroadcaster(cfg.DatabaseURL(), cfg.DatabaseListenerMinReconnectInterval(), cfg.DatabaseListenerMaxReconnectDuration(), appLggr, appID)
```

Comment:
Why is this under `services` now?

Reply:
Because of a circular dependency: services -> pg -> services

Reply:
Yeah so this is what I mentioned on the IPM, I ended up having a separate package because of this

Reply:
Right, and in order to fix this, I had to move EventBroadcaster out from pg. Is there a better idea to do this? Or do you believe EventBroadcaster should belong to pg?

Reply:
If you remove all services from the root `services` crate, then nothing will import `pg` there and the circular dependency will be gone

Reply:
All done